tools/proofers/ctrl_frame.php

```diff
@@ -9,9 +9,12 @@
 $round_id = get_enumerated_param($_GET, 'round_id', null, array_keys($Round_for_round_id_));
 $round = get_Round_for_round_id($round_id);
 
-$header_args = array(
-    "body_attributes" => 'onLoad="top.cRef = top.markRef = document.markform;" style="background-color: #CDC0B0; padding: 0; margin: 0;"',
-);
+$header_args = [
+    "js_files" => [
+        "$code_url/tools/proofers/character_selector.js",
+    ],
+    "body_attributes" => 'style="background-color: #CDC0B0; padding: 0; margin: 0;"',
```

Comment:
Please move this CSS into the `layout.less` and `theme.less` and then pull them in via:
```
"body_attributes" => 'class="control-frame'
```
or an ID or somesuch.